GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -843,7 +845,7 @@ private bool DeleteSelectedFiles()
 
         private void diffDeleteFileToolStripMenuItem_Click(object sender, EventArgs e)
         {
-            DeleteSelectedFiles();
+            DeleteSelectedFiles(deleteFiles: true, deleteFolders: false);
```

Comment:
The deletion should be what you delete, i.e. the menu item is only visible if all selected can be deleted.